app/modules/components/ModuleEdit.tsx

```diff
@@ -269,15 +269,23 @@ const ModuleEdit = ({
                         return <SearchResultModule item={item} />
                       },
                       noResults() {
+                        const matchedQuery = query.match(
+                          /^((http)s?:\/\/)?((dx\.)?doi\.org\/)?(10.\d{4,9}\/[-._;()\/:A-Z0-9]+$)/i
+                        )
+                        const matchedDoi = matchedQuery.slice(-1)
```

Comment:
I ended up assuming that the last element of the matched array is the DOI. I think a better approach would be to implement a look-behind in the regex, but I could not figure out the expression 😢

Reply:
Did you test that assumption? Either in a test version of the app or locally in your node cli.

Maybe the URLs from the Regex are a good start.

To consider merging this we'd need to be confident that this works 😊 I did a quick test run in my cli for one DOI and that one worked.

Reply:
I'm yet to log in to the app locally (still setting up and getting close), but I tested the logic on [Code Pen](https://codepen.io/nsunami/pen/YzEEKWM?editors=0011). It looks like it's working there. I'm happy to test them on my local once I get this working, but I need a bit more time.

Reply:
That's perfect, thanks @nsunami 😊 

I will do a local run, and the code pen looks promising.

Reply:
Great! I hope this works on the app!